pkg/lib/cockpit-components-password.jsx

```diff
@@ -106,19 +127,35 @@ export const PasswordFormFields = ({
                        validated={error_password ? "warning" : "default"}
                        id={idPrefix + "-pw1-group"}
                        fieldId={idPrefix + "-pw1"}>
-                <TextInput className="check-passwords" type="password" id={idPrefix + "-pw1"}
-                           autoComplete="new-password" value={password} onChange={(_event, value) => onPasswordChanged(value)}
-                           validated={error_password ? "warning" : "default"} />
-                <div>
-                    <Progress id={idPrefix + "-meter"}
-                              className={"ct-password-strength-meter " + variant}
-                              title={_("password quality")}
-                              size={ProgressSize.sm}
-                              measureLocation={ProgressMeasureLocation.none}
-                              variant={variant}
-                              value={Number.isInteger(passwordStrength) ? passwordStrength : 0} />
-                    <div id={idPrefix + "-password-meter-message"} className="pf-v5-c-form__helper-text" aria-live="polite">{passwordMessage}</div>
-                </div>
+                <InputGroup>
+                    <InputGroupItem isFill>
+                        <TextInput className="check-passwords" type={passwordHidden ? "password" : "text"} id={idPrefix + "-pw1"}
+                                   autoComplete="new-password" value={password} onChange={(_event, value) => onPasswordChanged(value)}
+                                   validated={error_password ? "warning" : "default"} />
+                    </InputGroupItem>
+                    <InputGroupItem>
+                        <Button
+                            variant="control"
+                            onClick={() => setPasswordHidden(!passwordHidden)}
+                            aria-label={ passwordHidden ? _("Show password") : _("Hide password")}>
+                            {passwordHidden ? <EyeIcon /> : <EyeSlashIcon />}
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19220-20230822-125449-71e7c813-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)